### PR TITLE
YugenMangas: Add ratelimit and increment connection timeout

### PR DESCRIPTION
**Pull request by bapeey**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
Is their site having troubles at the moment? Is it worth also adding a basic rate limit to the source of 1req/sec?

**Comment by bapeey** [comment]
> Is their site having troubles at the moment? Is it worth also adding a basic rate limit to the source of 1req/sec?

Yep you're right. Done

File: multisrc/overrides/heancms/yugenmangas/src/YugenMangas.kt [hunk 1]
```diff
@@ -19,6 +19,7 @@ class YugenMangas : HeanCms(
     override val client = super.client.newBuilder()
         .connectTimeout(60, TimeUnit.SECONDS)
         .readTimeout(90, TimeUnit.SECONDS)
+        .
         .build()
```
Comment (alessandrojean) [inline multisrc/overrides/heancms/yugenmangas/src/YugenMangas.kt:23]: Seems something is missing

Comment (bapeey) [reply multisrc/overrides/heancms/yugenmangas/src/YugenMangas.kt:23]: xD

**Comment by Clouddark75** [comment]
Their site has its CDN partially dead, probably because of the high concurrent user wave, or in the worst-case scenario, a big data loss, and probably because of that the site is hanging out with the invalid requests.

**Pull request merged**